core/domain/user_services.py

```diff
@@ -177,6 +177,7 @@ def get_users_settings(user_ids, strict=False, include_marked_deleted=False):
                 ],
                 banned=False,
                 username='admin',
+                user_has_viewed_lesson_info_once=False,
```

Comment:
How are we fetching 'user_has_viewed_lesson_info_once' in the explorations? I don't see this function being called anywhere while fetching exploration page data

Reply:
Yeah! That was a work in progress then. I have added them now. PTAL!